Scala/Scala.sublime-syntax

```diff
@@ -147,6 +147,46 @@ contexts:
     - match: '{{typeprefix}}'
       push: single-type-expression
 
+  annotation:
+    - match: '(@){{plainid}}\.'
```

Comment:
Sorry, missed this the first time: the `.` should be `punctuation.accessor.scala`.